CalculatorTests/CalculatorBrainTests.swift

```diff
@@ -0,0 +1,289 @@
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testConstants() {
+        let symbolsAndValues = ["π": Double.pi, "e": M_E]
+        for (symbol, value) in symbolsAndValues {
+            let result = returnConstantValue(symbol: symbol)
+            
+            XCTAssertEqual(result, value.roundTo(places: 6))
+        }
+    }
+    
+    func testUnaryFunctions() {
+        let unaryConstant: Double = 5
+        let unaryFunctionsAndAnswers = ["log": 1.609438, "sin": -0.958924, "cos": 0.283662, "tan": -3.380515, "√": 2.236068, "±": -5]
+        for (symbol,answer) in unaryFunctionsAndAnswers {
+            let result = returnUnaryOperationResult(unaryOperator: unaryConstant, symbol: symbol)
+            
+            
+            XCTAssertEqual(result, answer)
+        }
+    }
+    
+    func testBinaryFunctions() {
+        let firstOperator: Double = 7
+        let secondOperator: Double = 2
+        let binaryFunctionsAndAnswers = ["+": 9, "-": 5, "x": 14, "÷": 3.5]
+        
+//        let result = binaryFunctionsAndAnswers.reduce(0.0) { accumulator, operation in
+//            return returnBinaryOperationResult(firstOperator: accumulator, symbol: operation.key, secondOperator: operation.value)
+//        }
+        
+        for (symbol,answer) in binaryFunctionsAndAnswers {
+            let result = returnBinaryOperationResult(firstOperator: firstOperator, symbol: symbol, secondOperator: secondOperator)
+            
+            XCTAssertEqual(result, answer)
+        }
+    }
+    
+    func testConstantsDescription() {
+        let constants = ["e", "π"]
+        constants.forEach { constant in
+            
+            let result = returnConstantDescription(symbol: constant)
+            XCTAssertEqual(result, constant)
+        }
+    }
+    
+    func testUnaryDescription() {
+        let unarySymbols = ["log", "sin", "cos", "tan", "√", "±"]
+        
+        unarySymbols.forEach { symbol in
+            let description = "\(symbol)(5)"
+            let result = returnUnaryDescription(unaryOperator: 5, symbol: symbol)
+        
+            XCTAssertEqual(result, description)
+        }
+    }
+    
+    // Example found in task 7.a --- touching 7 + would show “7 + ...” (with 7 still in the display)
+    // FIXME Display is set at this level in the view controller as shown to do so in the lectures. Will refactor it after the second project task if he does not require it in setup.
+    func testExampleSevenA() {
+        var brain = CalculatorBrain()
+        brain.setOperand(7)
+        brain.performOperation("+")
+        let description = brain.description!
+        
+        XCTAssertEqual(description, "7+...")
+    }
+    
+    // Example found in task 7.b --- 7 + 9 would show “7 + ...” (9 in the display)
+    func testExampleSevenB() {
```

Comment:
Is this testing two things (description and evaluated answer)?

From the test names/values, I can't tell what the logic is (intended behaviour) for when the implementation should show "..." in the display, apart from in this specific case. Could the test be named in a way that explained the behaviour that it's asserting?